openedx/adg/lms/applications/forms.py

```diff
@@ -211,8 +211,6 @@ def clean(self):
         business_line_prerequisite = self.cleaned_data['business_line_prerequisite']
         is_common_business_line_prerequisite = self.cleaned_data['is_common_business_line_prerequisite']
 
-        if is_common_business_line_prerequisite and is_program_prerequisite:
+        if (business_line_prerequisite and (is_program_prerequisite or is_common_business_line_prerequisite)
+                or (is_common_business_line_prerequisite and is_program_prerequisite)):
```

Comment:
```
(is_program_prerequisite or is_common_business_line_prerequisite) or (is_common_business_line_prerequisite and is_program_prerequisite)
```
will always return value equal to `(is_program_prerequisite or is_common_business_line_prerequisite)` so lets just remove `or (is_common_business_line_prerequisite and is_program_prerequisite)`